CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+- Desktop: make dive sites 1st class citizens with their own dive site table
+- Desktop: only show dives at the dive sites selected in dive site table
+- Desktop: add undo functionality to edit operations and remove 'edited' state
```

Comment:
That's an anticipating entry, as the edited state is still there for cylinder and weight-system editing.

Reply:
true. I slightly overstated that. Feel free to edit / rewrite.